R/vec-utils.R

```diff
@@ -106,6 +106,16 @@ modify <- function(.x, ...) {
 #'
 #' seq2_along(10, letters)
 seq2 <- function(from, to) {
+  n_from <- length(from)
```

Comment:
This seems a bit weird to be since I thought `seq2()`  was `seq2(<integer>[1], <integer>[1])` — making it length 0 or 1 feels odd

Reply:
While not vectorised on its inputs, this function is used in vectorised context as it returns a vector. Perhaps I should implement this in a different unexported function so we can get a feel for it?

Reply:
I feel quite negatively towards it right now. The type feels very weird.